src/olympia/blocklist/tests/test_admin.py

```diff
@@ -21,6 +22,17 @@
 from ..models import Block, BlocklistSubmission
 
 
+IS_DJANGO_32 = django.VERSION[0] == 3
```

Comment:
Just wondering if you should actually call this `IS_DJANGO_3`, because that's all you're checking?

Reply:
It _would_ be more accurate to name it `IS_DJANGO_3`.  But we're only ever going to run this on 3.2 so doesn't really make any difference (we're skipping over 3.0 and 3.1 because they aren't LTS; the next release after 3.2 is 4.0).  Plus we can just drop this workaround once prod is running django3.2 and we drop the 2.2 tests.